WordPress/build.gradle

```diff
@@ -267,9 +270,8 @@ android {
         }
 
         // Also dynamically add additional `buildConfigFields` to our app flavors from any `wp.`/`jp.`-prefixed property in `gradle.properties`
```

Comment:
Minor (🔍): Consider update the comment to point to `default.properties` instead of `gradle.properties`.